doc/dev/cephfs-mirroring.rst

```diff
@@ -348,7 +348,7 @@ command:
 Commands that have the ``fs mirror status`` prefix provide mirror status for
 mirror-enabled file systems. Note that ``cephfs@360`` has the format
 ``filesystem-name@filesystem-id``. This format is required because mirror
-daemons are asynchronously notified of file-system mirror status (A file
+daemons are asynchronously notified of file-system mirror status (a file
```

Comment:
The whole "Commands that have the" sentence seems like a superfluous tautology to me and I suggest eliding it in toto.

Also, after prior furious debate, we try to converge on "file system" without an interstitial hyphen or space.  Even though right-thinking people would write "filesystem" ;)

>Note that the snapshot name in the above example has the format \`\`filesystem-name@filesystem-id\`\`.  This ensures consistent operation and reporting when, for example, a file system is deleted and recreated with the same name.